hack/e2e/run-rp-and-e2e.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash -e
 ######## Helper file to run E2e either locally or using Azure DevOps Pipelines ########
 
+set -x
```

Comment:
I'm not wild about this side-effect: any caller who sources this file and then handles secrets afterwards will leak their secret.

The PR should have all callers do `set -x` explicitly.